packages/core/components/Puck/index.tsx

```diff
@@ -436,13 +436,13 @@ export function Puck({
                                 />
                               </IconButton>
                             </div>
-                            <div>
+                            <>
                               {renderHeaderActions &&
                                 renderHeaderActions({
                                   state: appState,
                                   dispatch,
                                 })}
-                            </div>
+                            </>
```

Comment:
In this case we should wrap the <button> in the demo `renderHeaderActions` in a `<div>` instead of fragment `<>`, as otherwise the `<button>` is a direct flex child which cuases some layout issue in some responsive scenarioas (e.g. very long title text and/or slug).

Reply:
Here https://github.com/measuredco/puck/blob/6a56af016f0813e3a7eaea9c2d2448359709e93a/apps/demo/app/%5B...puckPath%5D/client.tsx#L63

Reply:
I've squashed that in